### PR TITLE
r2d2 lint config

### PR DESCRIPTION
**Pull request by RainThief**
The original commit in master includes config files already reviewed in component library.  only changes shown in this MR

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: React component library
+
+
+on:
+  push:
+    branches:
+      - "*"
```
Comment (lukaszraczylo) [inline .github/workflows/push.yml:7]: NIT: If the whole action depends on the merge into master we can specify master here.

The idea for future - from what I've gathered we can check if package.json has changed before allowing npm to publish new package ( as an attempt to publish a package with the same version will fail the pipeline ).

File: readme.md [hunk 2]
```diff
@@ -1,46 +1,56 @@
...
-This package provides the Royal Navy [ESLint](https://eslint.org/) configuration.
+# R2D2 Lint Config
+This package provides linting configuration to React Apps.
+
+@todo write readme
+@todo pipeline
 
 ## Installation
-The Royal Navy ESLint config is available as an NPM package.
+This is available as an NPM package or via got directly.
```
Comment (lukaszraczylo) [inline readme.md:8]: I am not sure about this sentence, something could be missing

File: readme.md [hunk 3]
```diff
@@ -1,46 +1,56 @@
...
-// npm
-npm install @royalnavy/eslint-config-react
+### NPM
+
+The package registry is on Github so you will need to apply a scoped registry to your `~/.npmrc`
 
-// yarn
-yarn add @royalnavy/eslint-config-react
+```
+@defencedigital:registry=https://npm.pkg.github.com/
```
Comment (lukaszraczylo) [inline readme.md:15]: NIT: We could add the cli friendly version

```
npm config set @defencedigital:registry https://npm.pkg.github.com/
```

File: .github/workflows/push.yml (hunk 1 above)
Comment (RainThief) [reply .github/workflows/push.yml:7]: on master their will always be a "patch" release so this should mean each push is a new increment

File: readme.md (hunk 3 above)
Comment (RainThief) [reply readme.md:15]: unfortunately GitHub package registry is as stable as the docker registry, a mixture of 40x errors!
therfore, as opensource components I am publishing to the main registry now
Any private packages will have to be installed via npm+git

**Review by lukaszraczylo — APPROVED** [review]

**Pull request merged**